360/watchapp/views.py

```diff
@@ -216,20 +216,8 @@ def homepage_view(request):
     friends = Friend.objects.friends(request.user)
 
     # Create a list of usernames from the friends queryset
-    friends_data = [{'username': friend.username, 'avatar': friend.profile.avatar.url} for friend in friends]
-
-
-    return render(request, 'watchapp/homepage.html', {"friends": friends_data})
-
-
-def logout_view(request):
-    # Clear the session data
-    request.session.flush()
-    # Redirect to the login page
-    return redirect('login')
-
-@login_required
-def settings_view(request):
```

Comment:
Just want to make sure we dont need this.

Reply:
I don't believe we do because I just got rid of the settings url and webpage and put all of its content onto the homepage. all of this code is still in the views page tho